Platforms/QemuQ35Pkg/PlatformPei/Platform.c

```diff
@@ -433,18 +450,22 @@ MiscInitialization (
     case 0xffff: /* microvm */
       DEBUG ((DEBUG_INFO, "%a: microvm\n", __FUNCTION__));
       MicrovmInitialization ();
-      PcdStatus = PcdSet16S (
-                    PcdOvmfHostBridgePciDevId,
-                    MICROVM_PSEUDO_DEVICE_ID
-                    );
+      PcdStatus = EFI_UNSUPPORTED;
+      // MU_CHANGE: Remove dynamic PCD set to support usage in Standalone MM
+      // PcdStatus = PcdSet16S (
+      //               PcdOvmfHostBridgePciDevId,
+      //               MICROVM_PSEUDO_DEVICE_ID
+      //               );
       ASSERT_RETURN_ERROR (PcdStatus);

```

Comment:
Maybe we never hit these cases, but won't this and the next one assert 100% of the time?

Reply:
Yes :) It is basically saying that we are not supporting types other than Q35.